www/js/main.js

```diff
@@ -50,6 +50,16 @@ angular.module('emission.main', ['emission.main.recent',
       }
     }
   })
+  
+  .state('root.main.leaderboard', {
+    url: '/leaderboard',
+    views: {
+      'leaderboard': {
+        templateUrl: 'templates/leaderboard.html',
+        controller: 'MetricsCtrl'
```

Comment:
why not have a separate controller for the leaderboard instead of glumping everything into `MetricsCtrl` which is big enough already?